packages/core/PluginLoader.ts

```diff
@@ -89,10 +88,6 @@ export default class PluginLoader {
     })
   }
 
-  moduleNameFromParsedUrl(parsedUrl: url.UrlWithStringQuery): string {
-    return 'fogbat'
-  }
-
```

Comment:
@rbuels I couldn't figure out how to get the module name from the parsed URL, since from what I could tell it was just something like `http://localhost:9000/plugin.js`, so I removed this function. If you can point me in the right direction of how it's supposed to work I can put it back in.

Reply:
yeah we don't know the url scheme yet, fine to remove